Lib/bisect.py

```diff
@@ -85,8 +81,11 @@ def bisect_left(a, x, lo=0, hi=None):
         else: hi = mid
     return lo
 
-# Overwrite above definitions with a fast C implementation
+# Overwrite above definitions with a fast Rust implementation
```

Comment:
In general you don't have to reword `C -> Rust` - there's a *lot* of mentions of C in the source code, so it's easier to just read "C" as "implementation language" than it is to change *all* mentions of C to rust. Applies to the test file as well.